Leanplum/Cache/LPCache.h

```diff
@@ -0,0 +1,41 @@
+//
+//  LPCache.h
+//  Leanplum
+//
+//  Created by Grace on 7/23/19.
+//  Copyright © 2019 Leanplum. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+@class LPVariable;
+@class LPRegion;
+@class LPMessage;
+@class LPVariant;
+@class LPUpdateRule;
+@class LPEventRule;
+@class LPVariantDebugInfo;
+
+@interface LPCache : NSObject
```

Comment:
can we make this a protocol and then have this class implement it?
That will help us change caching backings if needed